pkg/dbconfig/common.go

```diff
@@ -167,4 +178,18 @@ func (c *LoggingConfig) ConfigureLogging(spec *v1alpha1.LoggingSpec) {
 
 	c.Level = pointer.String(string(spec.Level))
 	c.LogFormat = pointer.String(string(spec.Format))
+
+	if spec.SlowQuery != nil {
+		c.EnableSlowQuery = pointer.Bool(spec.SlowQuery.Enabled)
+		c.SlowQueryThreshold = pointer.String(spec.SlowQuery.Threshold)
+
+		// Turn string to float64
+		ration, err := strconv.ParseFloat(spec.SlowQuery.SampleRatio, 64)
+		if err != nil {
+			klog.Warningf("Failed to parse slow query sample ratio '%s', use the default value 1.0", spec.SlowQuery.SampleRatio)
+			ration = 1.0
```

Comment:
We can set the default value of `ration = 1.0` in `defaulting.go`.

Reply:
It's already in `defaulting.go`.